packages/scrollable-image/package.json

```diff
@@ -23,9 +23,15 @@
   "dependencies": {
     "prop-types": "^15.0.0",
     "react": "^16.3.0",
+    "react-lazyload": "^2.6.5",
+    "react-waypoint": "^9.0.2",
+    "serialize-javascript": "^3.0.0",
     "styled-components": "^4.0.0"
   },
   "files": [
     "lib"
-  ]
+  ],
+  "peerDependencies": {
+    "uuid": "^7.0.2"
```

Comment:
move `uuid` to `dependencies`?

Reply:
BTW, please help to check all the `dependencies` and `devDependencies`.
For example, `webpack`, `webpack-cli` could be removed since webpack bundles are created at root (orangutan) folder.

Reply:
Sure, updated.